npm/webpack-dev-server/src/aut-runner.ts

```diff
@@ -21,22 +21,11 @@ export function init (importPromises, parent = (window.opener || window.parent))
   Cypress.onSpecWindow(window, importPromises)
   Cypress.action('app:window:before:load', window)
 
```

Comment:
The header is not what is hurting the dev tools

```suggestion
  // In this variable, we save head
  // innerHTML to account for loader installed styles
  let headInnerHTML = ''

  // before the run starts save
  Cypress.on('run:start', () => {
    headInnerHTML = document.head.innerHTML
  })
```